info.xml

```diff
@@ -14,20 +14,25 @@
     <url desc="Support">https://github.com/systopia/de.systopia.eventcheckin/issues</url>
     <url desc="Licensing">http://www.gnu.org/licenses/agpl-3.0.html</url>
   </urls>
-  <releaseDate></releaseDate>
+  <releaseDate/>
   <version>1.0-dev</version>
   <develStage>dev</develStage>
   <compatibility>
     <ver>5.0</ver>
```

Comment:
Is it possible that this needs to be raised to 5.45 or something similar? I think there was issues with the menu entries wih older versions, but idk which civix version requires which civicrm version.

Reply:
Or was it the upgrader....?